interface/patient_file/encounter/delete_form.php

```diff
@@ -38,7 +38,7 @@
 // if no custom 'delete' form, then use a generic one
 
 // when the Cancel button is pressed, where do we go?
```

Comment:
comment can be removed or changed (no options, but comment that "here is the reload target" is good)